README.md

```diff
@@ -1,22 +1,76 @@
+# SDS-in-a-box
 
-First you need to install nodejs (make sure its newer than version 14!)
+Hello!  This a project that attempts to capture the minimum requirements for the core of a Science Data System.  
 
-https://nodejs.org/en/download/
+Our goal with the project is that users will only need to modify the file config.json to define the data products stored on the SDC, and the rest should be mission agnostic.  
 
-Then you'll need to install the cdk:
+## Architecture
 
-npm install -g aws-cdk
+The code in this repository takes the form of an AWS CDK project. It provides the architecture for:
 
-pip install boto3
-pip install aws-cdk-lib
+1. An HTTPS API to upload files to an S3 bucket (*in development*)
+2. An S3 bucket to contain uploaded files
+3. An HTTPS API to query and download files from the S3 bucket (*in development*)
+4. A lambda function that inserts file metadata into an openseach instance
+5. A Cognito User Pool that keeps track of who can access the restricted APIs.  
 
-The first thing you'll need to do is configure your aws environemnt with 
 
+## Development
+
+The development environment uses a GitHub codespace, to ensure that we're all using the proper libraries as we develop and deploy.  
+
+Everyone gets 50 free hours per month of github Codespace time.  Alternatively, your organization can pay for it to run longer than this.  
+
+Top start a new development environment, click the button for "Code" in the upper right corner of the repository, and click "Codespaces".  
+
+
+### AWS Setup
+The first thing you'll need to do is configure your aws environemnt with:
+
+```
 aws configure
+```
+
+Enter in your AWS Access Key ID and AWS Secret Access Key, which can be obtained by setting up a user account in the AWS console. For region, set it to the AWS region you'd like to set up your SDS.  For IMAP, we're using "us-west-2"
 
-Then you'll need to bootstrap your cdk deployment with 
+**NOTE**-- If this is a brand new AWS account, then you'll need to bootstrap your account to allow CDK deployment with the command: 
 
+```
 cdk bootstrap
+```
+
+### Deploy
+
+Inside the app.py file, there are two important configuration items which you can alter:
+
+1) SDS_ID - This is just a string of 8 random letters that are appendend to each resources.  Alternatively, you can change this to something more meaningful (i.e. "bryan-testing").  Just be aware that this ID needs to be completely unique in each account.  
+
+To deploy the SDS, first you'll need to snyth the CDK code with the command:
+
+```
+cdk synth --context SDSID={insert a unique ID here}
+```
+
+and then you can deploy the architecture with the following command:
```

Comment:
```suggestion
This will generate the changes you are attempting to make, so you can review them if needed.  Then, you can deploy the architecture with the following command:
```

Just wanted to add a note about the difference between `synth` and `deploy`